docs/release/cut-sw-release.md

```diff
@@ -168,34 +182,30 @@ NON_UPCOMING_VERSIONS=<NON-UPCOMING VERSION(S)>
 
 dotest () {
     file=$dir/$client-$ver-1.$rhel.$arch.tar.gz
-    mkdir -p $rhel-$arch
-    pushd $rhel-$arch
-    tar xzf ../$file
-    $client/osg/osg-post-install
-    $client/osgrun osg-version
-    popd
-    rm -rf $rhel-$arch
+    if [ -e $file ]; then
```

Comment:
I'm not a big fan of the existence check. Presently, we expect 7 tarballs and if one of them doesn't exist, the tests should complain. If more than 7 tarballs are created, the tests should probably fail, too.

Reply:
I reworked the script to complain of missing tarballs. It still does not report extra tarballs. In any case, this is an improvement.